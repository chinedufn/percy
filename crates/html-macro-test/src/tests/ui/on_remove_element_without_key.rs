extern crate percy_dom;
use percy_dom::prelude::*;

// Used the `on_remove_element` attribute without providing a key attribute.
fn main() {
    html! {
        <div on_remove_element = ||{} >
        </div>
    };
}
