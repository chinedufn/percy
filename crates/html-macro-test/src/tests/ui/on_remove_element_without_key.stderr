error: Whenever you use the `on_remove_element=...` attribute,
you must also use must use the `key="..."` attribute.

Documentation:
  -> https://chinedufn.github.io/percy/html-macro/real-elements-and-nodes/on-remove-elem/index.html

 --> src/tests/ui/on_remove_element_without_key.rs
  |
  |         <div on_remove_element = ||{} >
  |              ^^^^^^^^^^^^^^^^^
